### ISSUE TITLE
Class discussion should be focused on current topic

### ISSUE THREAD

**Issue by joehand** [hackoregon/hack-university-data-architecture#2]
Keeping our discussion and decision making focused on the day's topic will improve our results and ensure everyone can participate equally.

We ended up discussing Hadoop for a large portion of the last class period, one that was supposed to be a retrospective. None of the previous projects used Hadoop. The discussion was something we need to have but it should not have been discussed yesterday.

It is unfair to class members who are unfamiliar with a technology to not have the opportunity to get a basic understanding of that technology before we engage in the discussion and start making decisions around it. If we are going to spend a majority of a class period discussing something, we should know before the session.

What can we do in the future to keep discussion focused around specific decision points or topics?

**Comment by pdxdiver** [comment]
Agreed.  I think we need to setup ground rules for facilitation. This means letting the topic get introduced or discussed for a couple of minutes,  but moving it to the backlog here in Issues.

**Comment by scottslewis** [comment]
I agree that the Hadoop discussion went on too long...especially since as Joe said we are going to dedicate future class time to exactly this technology and it's potential HO usage.  My apologies for allowing that.

It's my observation that this often happens in forward-looking architectural discussions, as people are appropriately enthusiastic about trying out new technologies and services.  

I suggest extending what pdxdiver proposed:   a short (e.g. max 15 minutes) introduction/discussion at the time.  Then a volunteer is found to create a bug and/or create wiki content describing the technology or service, so that others can get sufficient background on it, and it can (potentially) be scheduled for a future discussion/demo/trial/usage/etc.
